lte/gateway/docker/s1ap/build-s1ap.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# Copyright 2020 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+docker-compose -f docker-compose.yaml -f s1ap/docker-compose.override.yaml build --parallel s1aptester trfgen
```

Comment:
This script is expected to be called via `s1ap/build-s1ap.sh`, so from the viewpoint of the scripts the relevant files are in `../docker-compose.yaml` and if one would do `cd s1ap && ./build-s1ap.sh` it would fail. This is bad practice and the script should be made stable to work independent of the `cwd` of the caller. E.g. via

```suggestion
cd "$(dirname "$0")/.."
docker-compose -f docker-compose.yaml -f s1ap/docker-compose.override.yaml build --parallel s1aptester trfgen
```

Reply:
Fixed. Tested that I could run it from all folders and it worked.